WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -700,7 +700,6 @@ extension WordPressAppDelegate {
 
     @objc class func setLogLevel(_ level: DDLogLevel) {
         WPSharedSetLoggingLevel(level)
-        TracksSetLoggingLevel(level)
```

Comment:
In an internal Slack, @bjhomer pointed me towards the only likely breaking change between 0.9.x and 0.10.x, that is, that CocoaLumberjack has been removed.

The Tracks README also [explains](https://github.com/Automattic/Automattic-Tracks-iOS/tree/aaa60ea7e52e1bf6eb15f1511763d694566e91dc#logging) what to do to forwards logs coming from the library to the app: create an object conforming to [`TracksLoggingDelegate`](https://github.com/Automattic/Automattic-Tracks-iOS/blob/aaa60ea7e52e1bf6eb15f1511763d694566e91dc/Sources/Model/ObjC/Common/TracksLogging.h#L5-L13) and set it as `TracksLogging.delegate`. 

That should be rather straightforward, as the delegate has only a handful of methods, all of which can share the same implementation. I have two questions about how to implement it, though:

- What's the standard logging approach in WordPress iOS?
- Should we implement one at all, or go on without it? Has anyone looked at the logs searching for Tracks related info recently?

Reply:
> What's the standard logging approach in WordPress iOS?

I had a quick look around and I'm guessing CocoaLumberjack might be it. I decided to use it for the first pass at the delegate. See 0a011cc4073fad423cecb5897e35f1e2541cd327 and `TracksLogger` below.